app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -147,6 +147,12 @@ const withBackgroundEncryptedUpload = (Component) =>
       // eslint-disable-next-line react/jsx-props-no-spreading
       <Component {...props} onError={onError} onChange={onChangeWithBackgroundEncryptedUpload} />
     );
-  };
+  }
+
+  const { displayName: originalDisplayName = Component.name } = Component;
+  ComposedComponent.displayName = `WithBackgroundEncryptedUpload(${originalDisplayName})`;
```

Comment:
is this the ESLint way? I feel like for the one property, it might just be easier as:

```suggestion
  ComposedComponent.displayName = `WithBackgroundEncryptedUpload(${Component.displayName || Component.name})`;
```

Reply:
Mostly to avoid dealing with line length wrapping, since after Prettier it would become formatted as...

```suggestion
  ComposedComponent.displayName = `WithBackgroundEncryptedUpload(${
    Component.displayName || Component.name
  })`;
```

But yeah, in retrospect, the assignment line is pretty unwieldy, so maybe this suggestion is still the better option.

Reply:
Between those, I like the [line-broken version](https://github.com/18F/identity-idp/pull/5184#discussion_r661629168)

Reply:
> Between those, I like the [line-broken version](https://github.com/18F/identity-idp/pull/5184#discussion_r661629168)

Updated in 77e7ba8.